Sources/Model/TransferMethod/HyperwalletPayPalAccountQueryParam.swift

```diff
@@ -27,7 +27,7 @@ public class HyperwalletPayPalAccountQueryParam: HyperwalletTransferMethodQueryP
     override public func toQuery() -> [String: String] {
         var query = super.toQuery()
 
-        query[QueryParam.type.rawValue] = "PAYPAL_ACCOUNT"
+        query[QueryParam.type.rawValue] = type?.rawValue
```

Comment:
How are you going too restrict that only PAYPAL_ACCOUNT is set for this query param?